attestation-service/attestation-service/src/policy_engine/opa/mod.rs

```diff
@@ -2,40 +2,39 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
-use anyhow::{anyhow, bail, Context, Result};
+use anyhow::{Context, Result};
 use async_trait::async_trait;
 use base64::Engine;
 use sha2::{Digest, Sha384};
 use std::collections::HashMap;
 use std::fs;
 use std::path::PathBuf;
 
-use super::{PolicyDigest, PolicyEngine};
+use super::{OpaError, PolicyDigest, PolicyEngine};
```

Comment:
it's inconsistent, but should we maybe calling it "RegoError", since we are using regorus now and directly map its errors?